src/main/scala/rules/Brancher.scala

```diff
@@ -100,8 +100,8 @@ object brancher extends BranchingRules {
         }
 
         (v0: Verifier) => {
-          SymbExLogger.currentLog().switchToNextBranch(uidBranchPoint)
-          SymbExLogger.currentLog().markReachable(uidBranchPoint)
+          v.symbExLog.switchToNextBranch(uidBranchPoint)
```

Comment:
`v0` instead of `v`?

Reply:
My current understanding is that the verifier is carried through continuations only to enable branch parallelization, is that the case? If `v0 != v`, then the `symbExLog` of `v0` will be an incorrect `MemberSymbExLogger`, since the current member is accounted within one verifier. If there are cases when `v0 != v` other than when branch parallelization is enabled I guess we need some way to deal with that.

Reply:
Parallelisation is currently indeed the only motivation for threading the verifier through. But others are generally possible, e.g. instead of using state flags that control certain operations (such as extended-exhale), one could temporarily swap out algorithms, and these could be attached to the verifier.

But that's all hypothetical. It would be good, however, to document why `v` is used here, and what the expected relation between `v` and `v0` is.

Reply:
Taking that into account and after thinking about it a bit more I think `v0` is less wrong. Using `v` gives the impression that we will always use the original verifier for symbExLogging, but in fact we only delay the problem by ~1 log record if e.g. the verifier is (sequentially) swapped out. Consistently using the current verifier at least gives options, where you can exclude symbExLogging with the option (as with branch parallelization) or manually account the log in subverifiers.